src/Components/WebAssembly/Build/test/BuildIntegrationTests/BuildIntegrationTest.cs

```diff
@@ -276,6 +276,55 @@ public async Task Build_WithBlazorWebAssemblyEnableLinkingFalse_SatelliteAssembl
             Assert.Contains("ja/standalone.resources.dll", satelliteResources["ja"].Keys);
         }
 
+
+        [Fact]
+        public async Task Build_WithI8NOption_CopiesI8NAssembliesWithoutLinkerEnabled()
+        {
+            // Arrange
+            using var project = ProjectDirectory.Create("standalone", additionalProjects: new[] { "razorclasslibrary" });
+            project.Configuration = "Debug";
+            project.AddProjectFileContent(
+@"
+<PropertyGroup>
+    <BlazorWebAssemblyI18NAssemblies>other</BlazorWebAssemblyI18NAssemblies>
+</PropertyGroup>");
+
+            var result = await MSBuildProcessManager.DotnetMSBuild(project);
+
+            Assert.BuildPassed(result);
+
+            var buildOutputDirectory = project.BuildOutputDirectory;
+
+            Assert.FileExists(result, buildOutputDirectory, "wwwroot", "_framework", "_bin", "I18N.dll");
+            Assert.FileExists(result, buildOutputDirectory, "wwwroot", "_framework", "_bin", "I18N.Other.dll");
+            // When running without linker, we copy all I18N assemblies. Look for one additional
+            Assert.FileExists(result, buildOutputDirectory, "wwwroot", "_framework", "_bin", "I18N.West.dll");
+        }
+
+        [Fact]
+        public async Task Build_WithI8NOption_CopiesI8NAssembliesWithLinkerEnabled()
```

Comment:
Nit: I would change this function name to reflect the fact that the behavior is a little different without the linker enabled (e.g. assemblies that are not set in `BlazorWebAssemblyI18NAssemblies` are not copied.

Reply:
We already verify that the test does not copy additional assemblies when you have the linker enabled. The fact that we include additional assemblies when it isn't is an implementation defect, I don't think we need to document it as part of the test name. We could very well improve it in the future if we get feedback.